.config/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG grafana_version=latest
+ARG grafana_image=grafana-enterprise
```

Comment:
shouldn't we default to the community edition here?

Reply:
https://grafana.com/docs/grafana/latest/setup-grafana/installation/docker/#run-grafana-docker-image

> Note: The recommended and default edition of Grafana is Grafana Enterprise. It is free and includes all the features of the OSS edition. Additionally, you have the option to upgrade to the [full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page), which includes support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).

Reply:
Actually you're right.  `grafana-oss` is what I've been using locally.  I'll update this.